src/index.js

```diff
@@ -41,6 +41,9 @@ export function listen({
   // RethinkDB authKey for authenticated connections
   dbAuthKey = null,
 
+  // CA Certificate for the SSL connection, if required
```

Comment:
Done. :-)

On Mon, Feb 15, 2016 at 2:41 PM Mike Mintz <notifications@github.com> wrote:

> In src/index.js
> <https://github.com/mikemintz/rethinkdb-websocket-server/pull/11#discussion_r52936939>
> :
>
> > @@ -41,6 +41,9 @@ export function listen({
> >    // RethinkDB authKey for authenticated connections
> >    dbAuthKey = null,
> >
> > +  // CA Certificate for the SSL connection, if required
>
> This comment is a bit inaccurate now that we're passing in the whole
> object instead of just cacert. Can you change it to say something like "For
> TLS connections to RethinkDB, equivalent to ssl argument in r.connect"?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mikemintz/rethinkdb-websocket-server/pull/11/files#r52936939>
> .
>